### DISCUSSION TITLE
Can new options get backported into stable branch?

### DISCUSSION THREAD

**Discussion by cezdro** [nix-community/nixvim#1654] [Q&A] [answered]
My PR adding `luasnip.fromSnipmate` option has recently been merged, but I would like to switch back to the `nixos-24.05` branch, as I use that version of NixOS. I search through the commit log, but didn't see any previous example of such backport. Would it be ok, if I created a PR with cherrypicked commit?

**Comment by GaetanLepage** [comment, marked as answer]
We never backport changes to the stable branches. Now, if this specific commit would help you, go on and create a PR targetting the `nixos-24.05` branch. I don't see a reason not to merge it :)

**Comment by GaetanLepage** [comment]
(checkout the stable branch, cherry-pick the commit in question and push to your feature branch)
